examples/feast_feature_store/feast_feature_repo/example.py

```diff
@@ -7,7 +7,7 @@
 # production, you can use your favorite data warehouse, such as BigQuery. See Feast documentation
 # for more info.
 driver_hourly_stats = FileSource(
-    path="./data/driver_stats.parquet",
```

Comment:
What is this feast_feature_repo/ dir used for? Seems like it's not used in the example pipeline/steps/run.py code anywhere.

Reply:
@strickvl I am not too sure here, any idea

Reply:
It is used. There is a manual step that needs to be run from within that directory. It's needed :)